modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -1056,7 +1057,8 @@ protected int processStartTag() throws Exception {
 					resourceBundle, "x-results-for-x",
 					new Object[] {
 						getItemsTotal(),
-						(searchValue == null) ? StringPool.BLANK : searchValue
+						(searchValue == null) ? StringPool.BLANK :
+							HtmlUtil.escape(searchValue)
```

Comment:
For what is worth, this might be one scenario where what @csierra's team is working on (no decision still made as of last comment) would help if I understood correctly.

Reply:
Thanks @jbalsas I'll try to keep that in mind, next time I work on a similar issue.